docs/contributing.rst

```diff
@@ -62,6 +65,36 @@ can be used to test built assets:
 .. _Wyrm: http://www.github.com/snide/wyrm/
 .. _Sphinx: http://www.sphinx-doc.org/en/stable/
 
+
+_dockerized-build::
+
+Dockerized development
+======================
+
+If you have Docker available on your platform, you can get started building CSS and JS artifacts a bit faster and won't have to worry about any of the setup spilling over into your general environment.
+
+When building with Docker, we create an image containing the build dependencies. Some of these are quite outdated and therefore ideal to isolate a container. The image is tagged as ``sphinx_rtd_theme:latest``.
+
+Inside the running docker image, we mount the working copy of the repository, build the artifacts and finally observe that the artifacts have been built and left in your current git checkout.
+
+Use the following steps:
+
+.. code-block:: console
+
+    # Builds an updated version of the docker image
+    $ docker-compose build
+
+    # Runs the development webserver
+    $ docker-compose run sphinx_rtd_theme dev
```

Comment:
Wouldn't be better to use `docker-compose up` here which is the standard and call whatever is required behind the scenes?

Reply:
`docker-compose run` is doing what `docker run` does but reading the docker-compose configuration - I start the service that I want and give an argument `dev` to the entrypoint. `docker-compose up` is a different type of command.